src/app/GitCommands/Git/GitModule.cs

```diff
@@ -2388,6 +2389,7 @@ public async Task<ExecutionResult> GetGrepFileAsync(
             GitArgumentBuilder args = new("grep", commandConfiguration: commandConfiguration)
             {
                 "--line-number",
+                { showFunctionName, "--show-function" },
```

Comment:
0
This can be added in the options too, persistent
Implementation OK, will think if I want this or not